CHANNELS.md

```diff
@@ -0,0 +1,46 @@
+# Channels
+
+`fuelup` adopts a simplified version of `rustup` [channels](https://rust-lang.github.io/rustup/concepts/channels.html). Currently, only the `latest` channel will be published and serve as a source of distribution of Fuel toolchain binaries.
+
+The `latest` channel is possibly updated by a scheduled GitHub workflow that **runs every 30 minutes** and checks for new releases of `forc` and `fuel-core`.
```

Comment:
hmm, I used `possibly` because I wanted to say that the scheduled workflow doesn't always update the channel. Not sure if there's a better way to put this point across